packages/sources/por-address-list/src/endpoint/apiAddress.ts

```diff
@@ -0,0 +1,35 @@
+import {
+  PoRAddressEndpoint,
+  PoRAddressResponse,
+} from '@chainlink/external-adapter-framework/adapter/por'
+import { InputParameters } from '@chainlink/external-adapter-framework/validation'
+import { config } from '../config'
+import { apiAddressTransport } from '../transport/apiAddress'
+
+export const inputParameters = new InputParameters(
+  {
+    apiClient: {
+      required: true,
+      options: ['Bedrock uniBTC', 'SolvBTC'],
```

Comment:
Feel free to push back, but why not separate these to entirely separate endpoints? 

Then `endpoint` alone defines which is used rather than `endpoint` + `apiClient`, and the different protocols logic is kept entirely separate.

Reply:
I think there is still some level of grouping here, since both are simple API calls.

Similar to that we don't have one endpoint per chain in the regular por-address-list?

Reply:
I think the main distinction here is that these protocols both need separate request logic (to parse the response) - if they had the same logic then we could just have a generic `api` endpoint and pass the URL as a param, which would be equivalent to the chains in por-address-list.

`AddressTransport` isn't doing much, and I think the interface would be more obvious if you just needed `endpoint` to figure out which protocol the data is coming from (rather than `endpoint` + `apiClient`). There's also the nice side effect that adding new protocols is just adding a new transport, rather than changing code paths used by existing live feeds. 

Appreciate that these concerns aren't deal breakers, so I'm OK with going ahead with the current approach if you disagree with the above.

Reply:
I don't quite agree on logic separation but I take the point of two feeds impacting each other. If one API is slow it may impact the others. For errors I think we are covered under the catch all.

I will separate into two endpoints

Reply:
Split